asv_bench/benchmarks/pandas/join_merge.py

```diff
@@ -0,0 +1,165 @@
+import numpy as np
```

Comment:
Let's integrate these benchmarks into our system so we can see the difference and the new added cases.

Reply:
as for concat